src/ORM/FieldType/DBString.php

```diff
@@ -49,10 +54,10 @@ public function setOptions(array $options = []): static
     {
         parent::setOptions($options);
 
-        if (array_key_exists('nullifyEmpty', $options ?? [])) {
+        if (array_key_exists('nullifyEmpty', $options)) {
             $this->options['nullifyEmpty'] = (bool) $options['nullifyEmpty'];
         }
-        if (array_key_exists('default', $options ?? [])) {
+        if (array_key_exists('default', $options)) {
             $this->setDefaultValue($options['default']);
         }
```

Comment:
Note that previously `$this->options['default']` was being set to empty string - but `setDefaultValue()` wasn't being set! This put it in a weird situation where the default value in the database was an empty string (somehow - still not sure how that got there before), but calling `getDefaultValue()` would return `null`.